### PR TITLE
Simplify accept_tuple_argument decorator in space.py

### PR DESCRIPTION
**Pull request by Tortar**
This more readable decorator creates also less overhead

### CODE REVIEW

File: mesa/space.py [hunk 1]
```diff
@@ -391,7 +391,6 @@ def iter_cell_list_contents(
         # iter_cell_list_contents returns only non-empty contents.
         return (self.grid[x][y] for x, y in cell_list if self.grid[x][y])
 
-    @accept_tuple_argument
```
Comment (rht) [inline mesa/space.py:394]: I can merge without this line change. I think we can discuss this later.

Comment (Tortar) [reply mesa/space.py:394]: not a problem

Comment (Tortar) [reply mesa/space.py]: reverted

**Comment by rht** [comment]
LGTM

**Pull request merged**